### PR TITLE
confirm_modal.js

### PR DESCRIPTION
**Pull request by OlgaKhinevich**
Hello, I created new file for you)

### CODE REVIEW

File: confirm_modal.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+(function(){
+    function confirm_modal(text, options={}){
+        const {modal_type="default", header="Attention!"} = options;
+        if(!text) throw new Error("Modal text is required!");
+        const modal_wrapper = document.createElement("div");
+        modal_wrapper.className = "confirm_modal_wrapper";
```
Comment (bondcoder97) [inline confirm_modal.js]: Maybe we need to create common class "beauty_modal_wrapper" for both modal types.

File: confirm_modal.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                </div>
+                <div class="confirm_modal_btns">
+                    <span class="accept_btn btn">OK</span>
+                    <span class="cancel_btn btn">CANCEL</span>
+                </div>
+            </div>
+        `;
+        modal_wrapper.querySelector(".accept_btn").addEventListener("click", ()=> modal_wrapper.remove());
+        modal_wrapper.querySelector(".cancel_btn").addEventListener("click", ()=> modal_wrapper.remove());
+        document.body.appendChild(modal_wrapper);
```
Comment (bondcoder97) [inline confirm_modal.js:30]: Confirm modal need to return "true" or "false"!

**Review by bondcoder97 — CHANGES_REQUESTED** [review]
We need to edit some fragments.

**Comment by OlgaKhinevich** [comment]
Andrusha, I've done all tasks)

**Comment by bondcoder97** [comment]
Thanks! :)

**Pull request merged**